htdocs/compta/facture/class/facture-rec.class.php

```diff
@@ -347,7 +347,7 @@ public function create($user, $facid, $notrigger = 0)
 						{
 							// Extrafields
 							if (method_exists($facsrc->lines[$i], 'fetch_optionals')) {
-								$facsrc->lines[$i]->fetch_optionals($facsrc->lines[$i]->rowid);
+								$facsrc->lines[$i]->fetch_optionals($facsrc->lines[$i]->id);
```

Comment:
Are you sure the $fac_src->lines[$] is loaded with id.
Currently it seems to be loaded with fetch_array so property loaded is ->rowid (it should be stored into ->id), but it means this PR may break something.

Reply:
https://github.com/Dolibarr/dolibarr/blob/9e509a805a7d37b5cce3d306f881ebbed0e1dc64/htdocs/compta/facture/class/facture.class.php#L1699